wire-ios-data-model/Tests/Source/Model/User/AnalyticsIdentifierProviderTests.swift

```diff
@@ -19,14 +19,17 @@
 import XCTest
 @testable import WireDataModel
 
-class ZMUserTests_AnalyticsIdentifier: ModelObjectsTests {
+class AnalyticsIdentifierProviderTests: ModelObjectsTests {
```

Comment:
Same here! 🙌 

Can we make the class marked as final?

Reply:
done